app-doc/typst/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=typst
+PKGSEC=doc
+PKGDES="A new markup-based typsetting system that is designed to be as powerful as LaTeX while being much easier to learn and use."
```

Comment:
This is too verbose for a short description and it forces a lot of subjective descriptions. Try this...

A markup-based typsetting system

This should be good enough.